workspaces/liferay-dot-com-workspace/client-extensions/public-site-navigation-importer/src/main/resources/fragments/public-site-navigation/public-sites-navigation/index.html

```diff
@@ -115,7 +115,6 @@
 										siteNavigationMenuId=(configuration.accountMenuId)!0
 									/]
 								</div>
-
```

Comment:
@liaoethan  why was this removed? we should have empty lines between sibling dom elements.  - please run  `blade gw :formatSource` to fix that